src/redist/targets/GenerateBundledVersions.targets

```diff
@@ -95,6 +95,7 @@
           android-x64;
           android-x86;
           browser-wasm;
+          linux-musl-arm;
```

Comment:
I'm seeing the same thing above at line 86, maybe we're missing a piece of the puzzle here?

Reply:
https://github.com/dotnet/installer/commit/01930deb2a897ba302d89dacbe15a0622f2ecf0c cc @sdmaclea

Reply:
Maybe we just need to port that change to release?

Reply:
I see the build ID badge is missing in the readme.  I don't know if that is part of why the checksums aare missing from 5.0.1

Reply:
Ah, I had an older master where it was not present and haven't noticed it was actually there after the rebase. So I'll close the PR then and create a porting PR to the release. 
@ericstj what is the correct target branch for the porting PR? I am not familiar with the installer branch system.

Reply:
@sdmaclea or @marcpopMSFT would know this better, is it release/5.0.2xx or a yet to be created release/5.0.3xx?

Reply:
I am wondering if it might need to go on multiple branches.  We seem to release 3.1.109 & 3.1.403 simultaneously depending on which VS is installed.